interface/forms/misc_billing_options/new.php

```diff
@@ -34,110 +34,220 @@
+                        </div>
+                        <div class="form-group">
+                            <label class="form-inline"><?php echo xlt('Box 17. Provider') ?>:
+                                <?php  genProviderSelect('provider_id', '-- ' . xl("Please Select") . ' --', $obj{"provider_id"}); ?>
+                            </label>
+                            <label class="form-inline"><?php  echo xlt('Box 17. Provider Qualifier'); ?>:
+                                <?php  echo generate_select_list('provider_qualifier_code', 'provider_qualifier_code', $obj{"provider_qualifier_code"}, 'Provider Qualifier Code');?>
+                            </label>
+                        </div>
+                        <div class="form-group clearfix">
+                            <label for='hospitalization_date_from' class="col-sm-3 form-inline"><?php echo xlt('Box 18. Hospitalization date from');?>:</label>
+                                <?php $hospitalization_date_from = $obj{"hospitalization_date_from"}; ?>
+                                    <input type="text" class='datepicker col-sm-2 ' name='hospitalization_date_from' id='hospitalization_date_from' value='<?php echo attr($hospitalization_date_from); ?>' title='<?php echo xla('yyyy-mm-dd'); ?>'>
+                            <label for='off_work_to'class="col-sm-3 form-inline"><?php echo xlt('Box 18. Hospitalization date to');?>:</label>
+                                <?php $hospitalization_date_to = $obj{"hospitalization_date_to"}; ?>
+                                    <input type="text"  class='datepicker col-sm-2'  name='hospitalization_date_to' id='hospitalization_date_to' value='<?php echo attr($hospitalization_date_to); ?>' title='<?php echo xla('yyyy-mm-dd'); ?>'>
+                        </div>
+                        <div class="form-group">
+                            <label><?php echo xlt('Box 20. Is Outside Lab used?'); ?>:
+                                <input type="checkbox" name="outside_lab" value="1"
+                                <?php
+                                if ($obj['outside_lab'] == "1") {
+                                    echo "checked";
+                                }?>>
+                            </label>
+                            <label><?php echo xlt('Amount Charged'); ?>:
+                                <input type="text" size="7" align='right' name="lab_amount" value="<?php echo attr($obj{"lab_amount"});?>" >
+                            </label>
+                        </div>
+                        <div class="form-group">
+                            <label><?php echo xlt('Box 22. Medicaid Resubmission Code (ICD-10) ');?>:
+                                <input type="text"  name="medicaid_resubmission_code" value="<?php echo attr($obj{"medicaid_resubmission_code"});?>" >
```

Comment:
does this need an id of "medicaid_resubmission_code"?

Reply:
Ditto